package.json

```diff
@@ -20,7 +20,8 @@
     "start": "run-s clean sprites serve",
     "serve": "node scripts/serve.js",
     "build:code": "node scripts/build.js",
-    "build": "run-s clean sprites build:code taginfo status_map"
+    "build": "run-s clean sprites build:code taginfo status_map",
+    "test": "mocha test/spec/**/*.js"
```

Comment:
Glob needs to be quoted for portability. https://mochajs.org/#the-test-directory

But I'd just as soon do:

```sh
echo 'recursive: true' > .mocharc.yml
```

and then `mocha` can be invoked with no arguments.